src/config/shared.js

```diff
@@ -1 +1,34 @@
+import { createCatColorMap } from '../utils/color-maps';
+
 export const numHighlightedBuckets = 10;
+
+export const makeSortOrder = (latest, which = 'counts') => (a, b) => {
+  // get latest data point and see
+  if (latest[which][a] < latest[which][b]) return 1;
+  if (latest[which][a] >= latest[which][b]) return -1;
```

Comment:
The `=` is unnecessary, since 0 is returned otherwise.

```suggestion
  if (latest[which][a] > latest[which][b]) return -1;
```